site/_config.yml

```diff
@@ -120,6 +120,12 @@ exclude:
   - Runbook.docx
   - '*.sh'
   - vendor
+  - bundler/bundle/
+  - bundler/cache/
+  - bundler/gems/
+  - bundler/ruby/
```

Comment:
- Can we just exclude `bundler/`?
- Should we also exclude `.jekyll-cache`?

Reply:
Just excluding `bundler/` will not achieve will cause sub directories of `bundler` to be included in the build. The Jekyll link provided in the PR explains the details.

`.jekyll-cache` is new in Jekyll 4 and not present with the PR. It is ignored in the Jekyll4 PR.